test/e2e/data/cni/calico/calico.yaml

```diff
@@ -0,0 +1,792 @@
+---
```

Comment:
Can we also wget the CNI files from upstream repo?

Reply:
yep. changing that

Reply:
actually I didn't find the public hosted CNI file. let's keep the folder `data` as it since there are similar structure in CAPI, CAPV, CAPA etc